### ISSUE TITLE
reboot of the node when selinux is not set to disabled

### ISSUE THREAD

**Issue by gardoll** [ESGF/esgf-ansible#58]
**Describe the bug**

While upgrading ESGF nodes, esgf-ansible (4.0.0-beta1) has rebooted my data node during the task "selinux : Disable SELinux" because the SELinux policy of the node was set to "permissive" (which is not compliant with the ESGF settings spec, I admit). Following the FAQ page about the upgrade of existing nodes, ESGF services must be shutdown before starting esgf-ansible. But rebooting the node while running esgf-ansible, restarts the ESGF services (and most of the time, some of the services are not restarted - not deterministic -). This is due to the ansible task configuration:
```
TASK [selinux : Disable SELinux] 

...

    "policy": "targeted", 
    "reboot_required": true, 
    "state": "disabled"

    Config SELinux state changed from 'permissive' to 'disabled'
    [selinux : Pause for reboot confirmation]
```

**To Reproduce**

Set the SELinux to "permissive" (SELINUX=permissive in the /etc/selinux/config), reboot, stop ESGF services and start upgrading with esgf-ansible.

**Expected behavior**

I think esgf-ansible better terminate when dealing with machines with SELinux set to other values than "disabled", and output a clear message that SELinux must be manually set to "disabled".
Don't reboot the machine automatically.

**ESGF Node (please complete the following information):**
 - Managed OS: CentOS 6
 - Host OS: CentOS 6
 - esgf-ansible Version: 4.0.0-beta1
 - Node type: all

**Additional context**
Full context is described in the issue #48.

**Comment by nathanlcarlson** [comment]
Hi @gardoll,

As the next task is titled, it will pause to allow the user to confirm the reboot. The user may abort the deployment and take any action they would like on their own, then restart the deployment process. Additionally, it will only reboot if required. 

Did it fail to prompt you?

I did not consider the case where it started as permissive. This may trigger a reboot and cause problems as services will come back up upon reboot. In this case, a reboot should not be performed. Acceptable states should be `disabled` **or** `permissive`.

**Comment by gardoll** [comment]
I think it failed to prompt me if I would like to reboot or not because esgf-ansible didn't stop its processing. In fact, it was just to fast.

**Comment by nathanlcarlson** [comment]
I've discovered the cause (and other problems) and changes are being merged into `devel`. These changes will be part of the next release.

**Comment by nathanlcarlson** [comment]
The change made to resolve this can be found in #62. The summary is that Selinux is no longer being managed, so it will not reboot anything regarding that. Also, the use of `pause`, which would prompt the user, has been removed as it breaks when deploying to multiple machines at the same time.
